.travis.yml

```diff
@@ -23,11 +23,24 @@ install:
   - pip install .[complete]
   # Install extra requirements for running tests and building docs.
   - pip install -r requirements-dev.txt
+  - pip install sphinxcontrib-openapi
```

Comment:
Please delete this line and add it `requirements-dev.txt`.  This will help in this situation if someone is doing local development:

```sh
pip install -r requirements-dev.txt
make -C docs html  # would fail because sphinxcontrib-openapi is missing
```

It's also nice to have all the requirements in one place, instead of spread between `.travis.yml` and `requirements-dev.txt`. We make an exception for `doctr`, which is (correctly) installed separately below because `doctr` never makes sense to run locally, once it's configured.